bcgov/HOUS-permit-portal
# Cleanup PDF styles

## Description

Minor style fixes to accommodate new fonts along with refactor to use shared components with inline styles to make styles updates easier.

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

[BPHH-1074](https://hous-bssb.atlassian.net/browse/BPHH-1074)

## Steps to QA

1. Login as a submitter and submit a permit application that includes a checklist created with the auto-compliance tool.
2. Open the completed application and download the step code PDF (this may take a minute to generate)
3. Verify the step code PDF is generated properly and is styled appropriately.

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

n/a
